src/EditorFeatures/CSharpTest/MoveToNamespace/MoveToNamespaceTests.cs

```diff
@@ -16,14 +18,16 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.MoveToNamespace
     [UseExportProvider]
     public class MoveToNamespaceTests : AbstractMoveToNamespaceTests
     {
-        private static readonly IExportProviderFactory CSharpExportProviderFactory =
-            ExportProviderCache.GetOrCreateExportProviderFactory(
-                TestExportProvider.EntireAssemblyCatalogWithCSharpAndVisualBasic
-                    .WithPart(typeof(TestMoveToNamespaceOptionsService))
-                    .WithPart(typeof(TestSymbolRenamedCodeActionOperationFactoryWorkspaceService)));
+        private static readonly ComposableCatalog Catalog = TestExportProvider.EntireAssemblyCatalogWithCSharpAndVisualBasic;
```

Comment:
💡 This can be removed since you aren't reusing it across tests.

Reply:
Sure, I'll remove.